src/components/data/WdsTroubleshooter.js

```diff
@@ -51,11 +52,14 @@ export const WdsTroubleshooter = ({ onDismiss, workspaceId, mrgId }) => {
             setWdsStatus(res.status);
             setWdsDbStatus(res.components?.db?.status);
             setWdsPingStatus(res.components?.ping?.status);
+            // "Permissions" component only exists in WDS after 3da9bfc; be resilient
+            setWdsIamStatus(res.components?.Permissions ? res.components?.Permissions?.status : 'disabled');
```

Comment:
I think you could just do `setWdsIamStatus(res.components?.Permissions?.status ?? 'disabled');`